lesson13/main.js

```diff
@@ -0,0 +1,55 @@
+/*
+ Домашнее задание:
+В репозитории создать папку lesson_13
+В ней файл index.js
+В нем нужно написать несколько операций
+
+1.  Написать функцию “removeKeys1”, которая принимает два аргумента: первый это объект второй это массив строк.Эта функция должна вернуть исходный объект у которого не будет ключей, названия которых есть в массиве (втором аргумента). Например:removeKeys1({ a: 1, b: [], c: '' }, ['a', 'c']) Вернёт { b: [] }
+
+2.  Написать функцию “removeKeys2”, аналогичную первой, которая вернет новый объект, у которого будут те же ключи что и у первого аргумента, исключая те названия которых есть в массиве (втором аргументе)
+
+3.  Написать функцию “absDiff”, которая принимает аргумент “a”, и возвращает функцию, которая принимает только один аргумент аргумент “b”, и возвращает абсолютную разницу чисел “a” и “b”absDiff(3)(7) Вернёт 4
+
+Все написанные функции не должны быть вызваны ни разу в финальной версии кода!
+*/
+
+let o = {
+    a: 1,
+    b: [],
+    c: ""
+}
+
+let arr = ['a', 'c'];
+
+function removeKeys1(obj, arr) {
+    for (let key in obj) {
+        if (arr.includes(key)) {
+            delete obj[key];
+        }
+    }
+    return obj;
+}
+// removeKeys1(o, arr);
+
+
+function removeKeys2(obj, arr) {
+    let objNew = {
+        ...obj
+    }
+    for (let key in objNew) {
+        if (arr.includes(key)) {
+            delete objNew[key];
+        }
+    }
+    return objNew;
+}
+
+// removeKeys2(o, arr);
+
+function absDiff(numA) {
+    return (numB) => {
+        return Math.abs(numA - numB);
+    }
+}

```

Comment:
```suggestion
const absDiff = (numA) => (numB) => Math.abs(numA - numB);
```
Можно было бы написать так😉. Это как пример, необязательно исправлять